metrics/collectors/images/images_collector.py

```diff
@@ -92,6 +95,12 @@ def collect(self):
                 # the path ends with the image filename
                 image_name = path_table[-1]
 
+                image_type = None
+                for part_of_path in path_table:
+                    # If statement like this to cover daily-live, daily-legacy
+                    # daily-minimal and daily-preinstalled
+                    if "daily-" in part_of_path:
+                        image_type = part_of_path.replace("daily-", "")
```

Comment:
You can `break` here.

Looking at
```
RSYNC_SERVER_REQUESTS = [
    "rsync://cdimage.ubuntu.com/cdimage/daily*/*/*",
    "rsync://cdimage.ubuntu.com/cdimage/*/daily*/*/*",
    "rsync://cdimage.ubuntu.com/cdimage/*/dvd/*/*",
    "rsync://cdimage.ubuntu.com/cdimage/*/*/daily*/*/*",
    "rsync://cdimage.ubuntu.com/cdimage/*/*/dvd/*/*",
]
```
looks like the image "types" can be `daily*` or `dvd` probably deprecated. However if we want it more general we could do:
```
if part_of_path.startswith("daily") or part_of_path == "dvd":
    image_type = part_or_path
    break
```
this means keeping the `daily-` bit. Given that we also have "types" without "daily-", this is probably fine after all.